specs/Separate_conf_files.adoc

```diff
@@ -0,0 +1,141 @@
+= Separate out per-component configuration files
+
+Configuration details for the tendrl modules are read from same file(tendrl.conf).
+Based on the tendrl modules we have to keep the configuration details into different
+files with proper name and read it independently.
+
+== Problem description
+
+Currently all the configuration details for different modules are read from same file .
+We have to split the configuration details as per modules and keep that into different
+file with proper name. Each configuration file should have its own and relevant configuration
+information only.
+
+We have to modify the python file (config.py) which read configuration file as per this
```

Comment:
i will remove this points, i have already added this points in implementation section. These are duplicates so not required.